src/main/webapp/script.js

```diff
@@ -295,6 +322,15 @@ function createAuthenticationButton(authenticationUrl, buttonStyle, buttonText,
   authenticationButtonNavbar.appendChild(authenticationButtonItem);
 }
 
+/**
+ * Create message for the approval page that shows approval status directed to mentor or approver.
+ * TODO(oumontiel): Create content for message.
+ */
+function createApprovalMessage(example) {
```

Comment:
i can see that the 1, 2, and so on are in the example parameter, so you are just passing whatever you want as a message to the function, but you might want to consider doing all of the else-ifs over here so that you don't have many different types of call, perhaps passing the approval object as parameter

Reply:
Done!

Reply:
yeah, it is more readable now, thanks!